project/RELEASE-PROCESS.md

```diff
@@ -24,72 +24,36 @@ Building a Compose release
 
     Improvements to the code are not worth mentioning.
 
-5.   Add a bump commit:
-
-        git commit -am "Bump $VERSION"
-
-6.   Push the bump branch to your fork:
-
-        git push --set-upstream $USERNAME bump-$VERSION
-
-7.  Open a PR from the bump branch against the `release` branch on the upstream repo, **not** against master.
 
 ## When a PR is merged into master that we want in the release
 
-1.  Check out the bump branch:
+1. Check out the bump branch and run the cherry pick script
 
         git checkout bump-$VERSION
+        ./script/release/cherry-pick-pr $PR_NUMBER
 
-2.   Cherry-pick the merge commit, fixing any conflicts if necessary:
-
-        git cherry-pick -xm1 $MERGE_COMMIT_HASH
-
-3.  Add a signoff (it’s missing from merge commits):
-
-        git commit --amend --signoff
-
-4.  Move the bump commit back to the tip of the branch:
-
-        git rebase --interactive $PARENT_OF_BUMP_COMMIT
-
-5.  Force-push the bump branch to your fork:
+2. When you are done cherry-picking branches move the bump version commit to HEAD
 
+        ./script/release/rebase-bump-commit
         git push --force $USERNAME bump-$VERSION
 
-## To release a version (whether RC or stable)
 
-1.  Check that CI is passing on the bump PR.
+## To release a version (whether RC or stable)
 
-2.  Check out the bump branch:
+Check out the bump branch and run the `build-binary` script
 
         git checkout bump-$VERSION
+        ./script/release/build-binary
 
-3.  Build the Linux binary:
```

Comment:
Given these steps aren't automated yet, can we keep the docs in for how to build mac/windows binaries in?